apps/coa/views/proof-of-address.html

```diff
@@ -0,0 +1,40 @@
+{{<partials-page}}
+{{$page-content}}
+
+{{> partials-overview-header}}
+
+{{> partials-content-links}}
+
+{{#markdown}}address-intro{{/markdown}}
+
+
+<nav class="govuk-pagination govuk-pagination--block" role="navigation" aria-label="Pagination">
+  <div class="govuk-pagination__prev">
+    <a class="govuk-link--no-visited-state govuk-link--no-underline govuk-pagination__link" href="/proof-of-identity" rel="prev">
+      <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+        <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+      </svg>
+      <span class="govuk-pagination__link-title">
+        Previous<span class="govuk-visually-hidden"> page</span>
+      </span>
+      <span class="govuk-visually-hidden">:</span>
+      <span class="govuk-pagination__link-label">Proof of identity</span>
+    </a>
+  </div>
+  <div class="govuk-pagination__next">
+    <a class="govuk-link--no-visited-state govuk-link--no-underline govuk-pagination__link" href="/update-details" rel="next">
+      <svg class="govuk-pagination__icon govuk-pagination__icon--next" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+        <path d="m8.107-0.0078125-1.4136 1.414 4.2926 4.293h-12.986v2h12.896l-4.1855 3.9766 1.377 1.4492 6.7441-6.4062-6.7246-6.7266z"></path>
+      </svg>
+      <span class="govuk-pagination__link-title">
+        Next<span class="govuk-visually-hidden"> page</span>
+      </span>
+      <span class="govuk-visually-hidden">:</span>
+      <span class="govuk-pagination__link-label">Update your details</span>
```

Comment:
need to move those into translations, buttons.json

- Next
- Previous
- page
- Update your details